docs/docs/api_documentation_style.md

```diff
@@ -15,13 +15,13 @@ We employ the variants proposed by the [Google style](https://sphinxcontrib-napo
 
 We aim to achieve the following middle ground:
 
-- The documentation comments should be readable on the MATLAB command line when the `help FUN` command is used.
+- The documentation comments can be read on the MATLAB command line when the `help FUN` command is used, and it can be browsed through the `doc FUN` command.
```

Comment:
The use of the `doc` command niceties is likely to conflict with Sphinx. For example, `doc` does not recognize the Sphinx syntax for argument types or property types.

I bet that in Matlab 2019b, this works fantastically well with the new type validation blocks, see https://www.mathworks.com/help/matlab/matlab_prog/function-argument-validation-1.html ; in turn, this 2019b syntax is incompatible with both Octave and Matlab <2019b.

Accessibility is a key RepLAB value. (should we clarify those in a separate "values" document?)

Imho, our priorities should be:

1. Usable `help` documentation. This is the one that some users will call by default.

2. Usable Sphinx documentation, at least googlable. Other users will use this one (this is how I use YALMIP for example).

3. The `doc` command is convenient and we should accommodate it, but Sphinx should have priority.

4. Non-goal: breaking compatibility with Octave and earlier Matlab versions (we should define minimal supported Matlab versions. Perimeter Institute is still on Matlab 2015 something. Yalmip is developed on 2015a. We should support the Octave of the latest Ubuntu LSB release, right now 18.04).